syllabus/syllabus-app/src/webapp/syllabus/main.jsp

```diff
@@ -205,8 +205,8 @@
 					<f:verbatim></div></f:verbatim>
 					<f:verbatim>
 						<div>
-							<span id="successInfo" class="success popupMessage" style="display:none; float: left;"></span>
-							<span id="warningInfo" class="alertMessage popupMessage" style="display:none; float: left;"></span>
+							<span id="successInfo" class="sak-banner-success popupMessage" style="display:none; float: left;"></span>
+							<span id="warningInfo" class="sak-banner-error popupMessage" style="display:none; float: left;"></span>
```

Comment:
Should this be a warning (sak-banner-warn)?

Reply:
alertMessage messages are red, the replacement for red is sak-banner-error, right? Basically Signup only uses alertMessage for everything. I suggest merging this as red and change the colors later, analyzing what are errors and what are warnings.

Reply:
It is less about the colour and more about the meaning and intention of the banners.

Yes, having only one "alert" state for messages has been a problem in other tools too, so we've just had to use best judgment. We'll probably have to rewrite the alert system for these tools in the future to make it more flexible for our various banner types.

Because the span's id is "warningInfo", it makes sense to me (without looking at the message wording that populates these banners) to display the warning banner instead of error.